graphql/database/missions/completeMission.js

```diff
@@ -3,41 +3,45 @@ import getCurrentUserMission from './getCurrentUserMission'
 import {
   getPermissionsOverride,
   MISSIONS_OVERRIDE,
-  USERS_OVERRIDE,
 } from '../../utils/permissions-overrides'
 import MissionModel from './MissionModel'
 import {
   LONGEST_TAB_STREAK_AWARD_NAME,
   MOST_TABS_IN_DAY_AWARD_NAME,
   MOST_TOTAL_TABS_AWARD_NAME,
 } from './constants'
-import UserModel from '../users/UserModel'
 
 const missionsOverride = getPermissionsOverride(MISSIONS_OVERRIDE)
-const usersOverride = getPermissionsOverride(USERS_OVERRIDE)
 
 /**
  * @param {string} missionId
  * @param {string} userId
  */
 
-export default async (missionId, userId) => {
+export default async (userId, missionId) => {
   // get the mission with users
-  const mission = getCurrentUserMission(missionId, userId)
+  const mission = await getCurrentUserMission({
+    currentMissionId: missionId,
+    id: userId,
+  })
   if (mission == null) {
     return false
   }
   // verify that it is complete
   if (mission.tabCount < mission.tabGoal) {
     return false
   }
-
   // calculate end of mission awards
   const longestTabMember = mission.squadMembers.reduce((prev, current) =>
-    prev.longestTabStreak > current.longestTabStreak ? prev : current
+    prev.tabStreak.longestTabStreak > current.tabStreak.longestTabStreak
+      ? prev
+      : current
   )
   const mostDayTabsMember = mission.squadMembers.reduce((prev, current) =>
-    prev.missionMaxTabsDay > current.missionMaxTabsDay ? prev : current
+    prev.missionMaxTabsDay.maxDay.numTabs >
+    current.missionMaxTabsDay.maxDay.numTabs
+      ? prev
+      : current
```

Comment:
Please add tests for these.

Reply:
this one has tests, just had to update the data shape of the tests for it